### PR TITLE
Initial (un)version

### PR DESCRIPTION
**Pull request by SeanTAllen**

### CODE REVIEW

File: .github/workflows/changelog-bot.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    paths-ignore:
+      - CHANGELOG.md
+
+jobs:
+  changelog-bot:
+    runs-on: ubuntu-latest
+    name: Update CHANGELOG.md
+    steps:
+      - name: Update Changelog
+        uses: ponylang/changelog-bot-action@0.2.1
```
Comment (Theodus) [inline .github/workflows/changelog-bot.yml:14]: Shouldn't this use version 0.3.0?

File: RELEASE_PROCESS.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Start a release
+
+As documented above, a release is started by pushing a tag of the form `release-x.y.z`.
+
+## Announce release
+
+The release process can be manually restarted from here by push a tag of the form `announce-x.y.z`. The tag must be on a commit that is after "Release x.y.z" commit that was generated during the `Start a release` portion of the process.
+
+If you need to restart from here, you will need to pull the latest updates from the action-readme-version-updater repo as it will have changed and the commit you need to tag will not be available in your copy of the repo with pulling.
```
Comment (Theodus) [inline RELEASE_PROCESS.md]: "copy of the repo with pulling"? Do you mean "by pulling"?

File: .github/workflows/changelog-bot.yml (hunk 1 above)
Comment (SeanTAllen) [reply .github/workflows/changelog-bot.yml:14]: No. the latest version of changelog-bot is 0.2.1

File: RELEASE_PROCESS.md (hunk 2 above)
Comment (SeanTAllen) [reply RELEASE_PROCESS.md]: No. I meant "without pulling".

Comment (SeanTAllen) [reply RELEASE_PROCESS.md]: updated to be more clear.

**Review by Theodus — APPROVED** [review]
LGTM

**Pull request merged**